haystack/core/pipeline/pipeline.py

```diff
@@ -29,6 +29,7 @@
 
 from .descriptions import find_pipeline_inputs, find_pipeline_outputs
 from .draw import _to_mermaid_image
+from ... import tracing
```

Comment:
Strange this wasn't caught by the linter. We allow relative import from siblings (e.g. `from .draw import foo`) but we encourage absolute imports from parents.

Reply:
done